[2022-07-14 22:42:06,982] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: process_files.is_postgres_file_available manual__2022-07-14T22:42:05.826259+00:00 [queued]>
[2022-07-14 22:42:06,994] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: process_files.is_postgres_file_available manual__2022-07-14T22:42:05.826259+00:00 [queued]>
[2022-07-14 22:42:06,994] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-14 22:42:06,994] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-14 22:42:06,994] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-14 22:42:07,009] {taskinstance.py:1397} INFO - Executing <Task(HttpSensor): is_postgres_file_available> on 2022-07-14 22:42:05.826259+00:00
[2022-07-14 22:42:07,017] {standard_task_runner.py:52} INFO - Started process 1258 to run task
[2022-07-14 22:42:07,020] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'process_files', 'is_postgres_file_available', 'manual__2022-07-14T22:42:05.826259+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/process_files.py', '--cfg-path', '/tmp/tmp1zq4et89', '--error-file', '/tmp/tmpjxo96v51']
[2022-07-14 22:42:07,020] {standard_task_runner.py:80} INFO - Job 7: Subtask is_postgres_file_available
[2022-07-14 22:42:07,086] {task_command.py:371} INFO - Running <TaskInstance: process_files.is_postgres_file_available manual__2022-07-14T22:42:05.826259+00:00 [running]> on host 4bf3acff5795
[2022-07-14 22:42:07,183] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=process_files
AIRFLOW_CTX_TASK_ID=is_postgres_file_available
AIRFLOW_CTX_EXECUTION_DATE=2022-07-14T22:42:05.826259+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-14T22:42:05.826259+00:00
[2022-07-14 22:42:07,184] {http.py:97} INFO - Poking: world.sql
[2022-07-14 22:42:07,196] {base.py:68} INFO - Using connection ID 'http_api' for task execution.
[2022-07-14 22:42:07,198] {http.py:129} INFO - Sending 'GET' to url: http://github.com/tadinve/EKG-Foundations/blob/master/04A-postgress/world.sql
[2022-07-14 22:42:07,905] {base.py:301} INFO - Success criteria met. Exiting.
[2022-07-14 22:42:07,926] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=process_files, task_id=is_postgres_file_available, execution_date=20220714T224205, start_date=20220714T224206, end_date=20220714T224207
[2022-07-14 22:42:07,954] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-14 22:42:08,005] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
